### PR TITLE
I reforked the repo to make necessary changes

### PR DESCRIPTION
**Pull request by brownfrosamurai**
I had some problems with the previous repo soo I deleted it and forked the repo.

I added the HTML files for the landing page, the product page as well as the sign-up page ...
landingpage.html,
productpage.html,
sign-up.html

I added styling for the HTML files
style.css,
productstyle.css,
landingstyke.css.

I added a js folder which contains the script.js file 

I also added the solution to the UX design assignment.
and Balsamiq wireframes for the e-commerce site.

I added a folder which contains all the images used in the design.

### CODE REVIEW

File: submissions/meduna oluwafemi/HTML/JS folder/script.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        price:"$400"
+
+    },
+    {
+        name:"Nike-urban" ,
+        imageUrl: "./img/image 3.webp",
+        price:"$135"
+
+    }
+
```
Comment (beccadaniel) [inline submissions/meduna oluwafemi/HTML/JS folder/script.js]: @brownfrosamurai  You might want to make sure that you don't have unnecessary line spaces in your files... There are a couple of them in your submissions. Do check it out

File: submissions/meduna oluwafemi/HTML/css/productstyle.css [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+* {
+    margin: 0;
+    padding: 0%;
+    font-family:'Poppins', sans-serif; 
+    box-sizing: border-box;
+    text-decoration: none;
+
+}
+
+
```
Comment (beccadaniel) [inline submissions/meduna oluwafemi/HTML/css/productstyle.css]: Same comment as the previous one. Ensure you do not have unnecessary line spacing in your files.

File: submissions/meduna oluwafemi/HTML/landingpage.html [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+</div>
+
+<div class="banner-3">
+    <h2>casual fashion</h2>
+    <p><a href="#">shop</a> </p>
+</div>
+    <div class="product-title2"><p>more to explore</p></div>
+    
+    <div class="products-container">   
+    <div class="products">
```
Comment (beccadaniel) [inline submissions/meduna oluwafemi/HTML/landingpage.html:98]: @brownfrosamurai  Ensure that you use proper indentation. For instance, your closing div indentation should be the same as the indentation of the starting div... and others like that.
But this is good work, well done!!!

**Review by beccadaniel — CHANGES_REQUESTED** [review]

File: .vscode/settings.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (kenny-wealth) [inline .vscode/settings.json]: We don't want to check this in. Please add `.vscode` to gitignore

File: submissions/meduna oluwafemi/HTML/JS folder/script.js (hunk 1 above)
Comment (brownfrosamurai) [reply submissions/meduna oluwafemi/HTML/JS folder/script.js]: I have resolved the issues.

File: .vscode/settings.json (hunk 4 above)
Comment (brownfrosamurai) [reply .vscode/settings.json]: I have resolved the issue.

**Review by beccadaniel — APPROVED** [review]

**Review by tegaphilip — APPROVED** [review]

**Pull request merged**